packages/protocol/contracts/governance/Validators.sol

```diff
@@ -1131,4 +1147,57 @@ contract Validators is
       }
     }
   }
+
+  /**
+   * @notice Sets the slashingMultiplierRestPeriod property if called by owner.
+   * @param value New reset period for slashing multiplier.
+   */
+  function setSlashingMultiplierResetPeriod(uint256 value) public nonReentrant onlyOwner {
+    slashingPenaltyResetPeriod = value;
```

Comment:
Ditto, inconsistency. I know I'm probably responsible for suggesting `penalty` in the first place. I think we had previously talked about `rewardsMultiplier`, which I don't like and would conflict with a term we use in `EpochRewards.sol`. `slashingMultiplier`, however, is fine IMO. Happy with w/e you choose.